docs/migration/4_0.md

```diff
@@ -1,3 +1,35 @@
 # Technical Changes in Spartacus 4.0
 
 ## Breaking Changes Introduced in 4.0
+
+## Automated Migrations for Version 4.0
+
+- `ProductVariantsModule` was removed from @spartacus/storefront. Use `@spartacus/product/variants` feature-library instead.
```

Comment:
Duplicate of line 9. I would remove this one

Reply:
I removed second occuring of this, as it's more misleading with container part